### PR TITLE
style: spacing for chat-body button - issue #73

### PR DESCRIPTION
**Pull request by jazzninja**
Update per issue #73 direction and request for chat-body button pending links spacing

### CODE REVIEW

File: src/styles.css [hunk 1]
```diff
@@ -310,7 +310,7 @@ a:active {
 }
 
 .chat li .chat-body p {
-   margin: 0;
```
Comment (ChrisDinhNZ) [inline src/styles.css]: @jazzninja , Line 313 does not need to be changed.

File: src/styles.css [hunk 2]
```diff
@@ -310,7 +310,7 @@ a:active {
...
+   margin: 0 5px;
 }
```
Comment (ChrisDinhNZ) [inline src/styles.css]: Can you add the following:

> .chat li .chat-body button {
>    margin: 0 5px;
> }

between line 314 and line 316?

Comment (jazzninja) [reply src/styles.css]: yup

**Comment by jazzninja** [comment]
New PR has been made. Let me know if this was what you were looking for or if you need anything from me

File: src/styles.css (hunk 2 above)
Comment (jazzninja) [reply src/styles.css]: @ChrisDinhNZ I've updated and pushed the edits to the same branch. I'd opened a new PR in error for the change/update you requested, but just closed it out. Let me know if this branch update is good to go for merge, thanks.

Comment (ChrisDinhNZ) [reply src/styles.css]: Thanks for that.

**Pull request merged**